src/sensors/icm20948sensor.h

```diff
@@ -41,19 +41,31 @@ class ICM20948Sensor : public Sensor
 
     void sendData() override final;
     void startCalibration(int calibrationType) override final;
-    void save_bias(bool repeat);
-    void load_bias();
+    void saveCalibration(bool repeat);
+    void loadCalibration();
+
+    void startCalibrationAutoSave();
+    void startDMP();
+    void connectSensor();
+    void startMotionLoop();
+    void checkSensorTimeout();
+    void checkForDataToRead(ICM_20948_Status_e readStatus);
+    void readRotation();
+    bool checkIfDataToRead();
```

Comment:
most of these should be private, probably

Reply:
Your right, they should.